clients/drcachesim/scheduler/scheduler.h

```diff
@@ -833,6 +840,9 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
         bool recorded_in_schedule = false;
         // This is a per-workload value, stored in each input for convenience.
         uint64_t base_timestamp = 0;
+        // This equals 'options_.deps == DEPENDENCY_TIMESTAMPS', stored here for
+        // access in InputTimestampComparator.
+        bool order_by_timestamp = false;
```

Comment:
The comparator only has static access to what's stored in the queue which are input_info_t's.  I guess an alternative is to store a `this` pointer to the scheduler_t instead of this bool?

Reply:
I think I like this better than storing a schedule_t parent pointer here.  Augmented the comment to explain a little more.  Not ideal but I don't think we have a better option.